src/data/set/pointwise/basic.lean

```diff
@@ -1222,6 +1222,22 @@ begin
     exact ⟨b, mem_inter (mem_smul_set.mpr ⟨a, ha, by simp [← this]⟩) hb⟩, },
 end
 
+@[simp, to_additive] lemma Union_inv_smul :
+  (⋃ (g : α), g⁻¹ • s) = (⋃ (g : α), g • s) :=
+begin
+  ext a,
+  simp only [mem_Union],
+  split;
+  rintros ⟨g, hg⟩,
+  { exact ⟨g⁻¹, hg⟩, },
+  { exact ⟨g⁻¹, by rwa inv_inv⟩, },
+end
```

Comment:
This ought to be a one-liner from `equiv.supr_congr` (or more precisely from its missing `set.Union` counterpart).

Reply:
Thanks, I've golfed it (using `equiv.supr_congr`).